src/features/query-bar/components/filter/FilterButton.js

```diff
@@ -69,13 +69,13 @@ class FilterButton extends React.Component<Props, State> {
         const { columns } = this.props;
         if (columns) {
             const firstColumn = columns[0];
-            const operator = COLUMN_OPERATORS[firstColumn.type][0].key;
+            const operator = [COLUMN_OPERATORS[firstColumn.type][0].key];
```

Comment:
Is it possible for Condition to support multiple operators? If not, let's go this route.